test/tap/tests/test_ps_async-t.cpp

```diff
@@ -220,7 +384,7 @@ int main(int argc, char** argv) {
 	// configure the connection as not blocking
 	diag("Setting mysql connection non blocking");
 	mysql_options(mysql, MYSQL_OPT_NONBLOCK, 0);
-	if (!mysql_real_connect(mysql, cl.host, cl.username, cl.password, NULL, cl.port, NULL, 0)) {
+	if (!mysql_real_connect(mysql, cl.host, cl.username, cl.password, NULL, 6033, NULL, 0)) {
```

Comment:
This change should be reverted I believe

Reply:
Whoops, yes, thanks you.